bothan-binance/src/worker/asset_worker.rs

```diff
@@ -22,18 +22,14 @@ pub(crate) async fn start_asset_worker(
     mut subscribe_rx: Receiver<Vec<String>>,
     mut unsubscribe_rx: Receiver<Vec<String>>,
 ) {
-    loop {
+    while let Some(worker) = worker.upgrade() {
```

Comment:
Upon asset worker starting, does it need to subscribe to whatever already exists in the store's `query_ids`?

Reply:
resolved in build()